patterns/product-collections-newest-arrivals.php

```diff
@@ -9,37 +9,39 @@
 
 <!-- wp:group {"align":"wide","layout":{"type":"constrained"}} -->
 <div class="wp-block-group alignwide">
-	<!-- wp:group {"align":"full","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
-	<div class="wp-block-group alignfull">
-		<!-- wp:heading {"level":1,"style":{"typography":{"fontSize":"48px"}}} -->
-		<h1 class="wp-block-heading" style="font-size:48px">Our newest arrivals</h1>
-		<!-- /wp:heading -->
+	<!-- wp:columns {"align":"wide"} -->
+	<div class="wp-block-columns alignwide">
+		<!-- wp:column -->
+		<div class="wp-block-column">
+			<!-- wp:heading {"level":3} -->
+			<h3 class="wp-block-heading">Our newest arrivals</h3>
```

Comment:
Should strings in patterns be translatable? I'm making them translatable in the patterns I edit, but wondering if I should not! :sweat_smile:

Reply:
That's a good question! I would vote for having them translatable, it's little effort for a positive impact for international users.

Reply:
Agree!